play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/GenerateResources.kt

```diff
@@ -52,6 +53,7 @@ internal abstract class GenerateResources @Inject constructor(
     abstract val resSrcDirs: ListProperty<Directory>
 
     @get:SkipWhenEmpty
+    @get:IgnoreEmptyDirectories
```

Comment:
Something internally has changed on Gradle 8.1.1 and `IgnoreEmptyDirectories` now is required to keep getting `NO-SOURCE` outcome on the tests